layers/brent.map

```diff
@@ -153,4 +153,40 @@ MAP
     END
   END
 
+  LAYER
+    NAME "Highway_Verges"
+    METADATA
+      "wfs_title"         "highway_verges" ##REQUIRED
+      "wfs_srs"           "EPSG:3857 EPSG:27700" ## REQUIRED
+      "gml_include_items" "Roadname"
+      "gml_featureid"     "Roadname" ## REQUIRED
```

Comment:
Now I see this I wonder if it might be easier if these two new layers use `site_name` to be consistent with the existing parks and housing layers.

You can do that with:

```
"gml_Roadname_alias" "site_name"
```

And same for highway verges below (except the key will be `gml_name_alias` for that).